CMakeLists.txt

```diff
@@ -585,6 +585,11 @@ elseif(WIN32)
     target_link_libraries(antimicrox
         ${WIN_LIBS}
     )
+
+    option(PORTABLE "Compile app for portability (Windows only)" OFF)
```

Comment:
This option should be moved at the beginning of Cmake file (next to other options) and rename it to something like PORTABLE_WIN

```cmake
if(WIN32)
    option(PORTABLE_WIN "Compile app for portability" OFF)
endif(WIN32)
```

Reply:
Done.